src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1144,6 +1144,33 @@ class VogelIPRDescription(CommonIPR):
     # [[[end]]] (checksum: 9934660f6467a25e11bfbe4c95f8faa7)
 
 
+@attr.s(frozen=True, slots=True)
+class UndersaturatedOilIPRDescription(CommonIPR):
```

Comment:
As I suggested in the alfasim PR. Please, consider renaming this class to `FetkovichIPRModel`.

Reply:
Done